### PR TITLE
add Harsh's hackbar

### PR DESCRIPTION
**Pull request by harshavardhan-i**
Thanks for initiating a project for beginners.

### CODE REVIEW

**Comment by harshavardhan-i** [comment]
I'm not sure if I've to make the "hactoberfest" labels or is it done by you!
This is my first ever PR on an open source project

**Comment by tasnimzotder** [comment]
@gugacavalieri, I'll add the `hacktober-accepted` label after the approval of the PR. Moreover, no need to add `hacktoberfest` label to this PR as the project is under `hacktoberfest` topic.

File: _hackbars/harsh_i.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+layout: hackbar
+title: Open Source
+author: Harsh I
+---
```
Comment (tasnimzotder) [inline _hackbars/harsh_i.md:5]: hey @harshavardhan-i, need to type a more line code 👨‍💻

```suggestion
---
layout: hackbar
title: Open Source
author: Harsh I
---
```

File: _hackbars/harsh_i.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+This is first ever open source PR. I think it's a good start.
+
+---
+
+## The global rise in temprature has pushed a lot of species to extinction 
+
+![Draught](/hacktoberfest-2020/assets/images/harsh_i.jpg)
```
Comment (tasnimzotder) [inline _hackbars/harsh_i.md]:
```suggestion
![Draught]({{site.baseurl}}/assets/images/harsh_i.jpg)
```

**Review by tasnimzotder — APPROVED** [review]

**Comment by tasnimzotder** [comment]
Thanks for your contributions :)

**Pull request merged**